aspnetcore/signalr/diagnostics.md

```diff
@@ -100,6 +100,8 @@ If you want to send logs to a custom logging system, you can provide a JavaScrip
 
 To get logs from the .NET client, you can use the `ConfigureLogging` method on `HubConnectionBuilder`. This works the same way as the `ConfigureLogging` method on `WebHostBuilder` and `HostBuilder`. You can configure the same logging providers you use in ASP.NET Core. However, you have to manually install and enable the NuGet packages for the individual logging providers.
 
+To add .NET client logging to a Blazor WebAssembly app, see <xref:fundamentals/logging/index#blazor=webassembly-signalr-net-client-logging>.
```

Comment:
```suggestion
To add .NET client logging to a Blazor WebAssembly app, see <xref:fundamentals/logging/index#blazor-webassembly-signalr-net-client-logging>.
```

Reply:
Thanks ... strange that the build system didn't catch that.